### ISSUE TITLE
Error after changing Bitwarden_rs master password and encryption

### ISSUE THREAD

**Issue by Simmelinus** [dani-garcia/bitwarden_rs#1240]
**Error after changing Bitwarden_rs master password and encryption**

Hi,

today I changed the master password of my Bitwarden_rs and I also checked the box to change the encryption.
As Bitwarden_rs announced before I was logged off after that of course. But beside the note that I was logged off, there were two red error boxes under the log out message on the upper right side of my screen as well. Both just said that an error occured. Just one sentence. No error code or additional information. Is this OK and part of the routine or not?

I tried to log in with the new master password and everything works fine as it was before. But still I am a little bit concerned that maybe something with the encryption is not right and that there could be a security issue with my password encryption. Could this be the case and how can I find out?

My environment:
My Bitwarden_rs installation runs on a Raspberry Pi 3 B in a Docker container (installed via image that I got from Docker) and with a traefik reverse proxy.
My Bitwarden_rs version is 1.17.0.
I use Bitwarden_rs with the Andoid App as well as the Chrome extension. But when the error occured I was logged in via my DDNS domain on Chrome.

I reproduced the change of the master password one more time (again with the checked box for the encryption) and the two erros again occured.

In summary I need to know if these errors are normal or not - and if not: What could be the reason and what needs to be done to make sure that my password are safe and ecrypted.

Thank you!

**Comment by BlackDex** [comment]
@Simmelinus, i just checked it, and this should not happen i think. But i can reproduce it.
Using vault.bitwarden.com this doesn't happen, so there probably is an issue on our side.

**Comment by BlackDex** [comment]
Thanks for reporting, i have fixed this in a PR.

**Comment by Simmelinus** [comment]
@BlackDex  Thanks a lot for your quick reaction! Was this issue security-related and is it necessary to change my passwords that I store in Bitwarden_rs or is everything safe?

**Comment by BlackDex** [comment]
Well basically yes, but it was a more strict security which caused this :). So no issues. I only think that you maybe indeed should do a rotate again using the new version when the PR is merged. Because the new changed rotated keys arn't used/saved correctly now.

**Comment by Simmelinus** [comment]
Alright, great. Thank you!
